packages/utils/src/cache/storages/localstorage.ts

```diff
@@ -0,0 +1,125 @@
+import type { CacheMap } from '../types';
+import type { CacheItem } from '../interfaces/CacheItem';
+import type { CacheStorage } from '../interfaces/CacheStorage';
+import { PREFIX } from '../constants.js';
+
+/**
+ * Stores data in `localStorage` for use across multiple sessions.
+ */
+export class CacheLocalStorage implements CacheStorage {
+
+  /**
+   * Prefix for all keys
+   * to avoid key to clash with the workspace
+   */
+  protected prefixKey = '';
```

Comment:
I think this should be align with indexdb storage class.

Reply:
Updated.
I create a helper getItemKey for a key in localstorage. The helper will be replace logic of handle key. And change prefixKey to dbName.